cerberus-web/src/main/java/com/nike/cerberus/service/AuthenticationService.java

```diff
@@ -186,6 +186,30 @@ public AuthResponse triggerChallenge(final MfaCheckRequest challengeRequest) {
     return authResponse;
   }
 
+  /**
+   * Enables a user to trigger a factor challenge.
+   *
+   * @param challengeRequest Request containing the MFA token details with no passcode
+   * @return The auth response
+   */
+  public AuthResponse triggerPush(final MfaCheckRequest challengeRequest) {
+    final AuthResponse authResponse =
+        authServiceConnector.triggerPush(
+            challengeRequest.getStateToken(), challengeRequest.getDeviceId());
+    if (authResponse.getStatus() == AuthStatus.SUCCESS) {
+      authResponse
+          .getData()
+          .setClientToken(
+              generateToken(
+                  authResponse.getData().getUsername(),
+                  authServiceConnector.getGroups(authResponse.getData()),
+                  0));
+    } else {
```

Comment:
Should this else block be empty?

Reply:
Nope, that got caught from a copy past. Nice catch.